### PR TITLE
Feature/Multiple-node-support part of #40

### PR DESCRIPTION
**Pull request by Senna46**
@YasunoriMATSUOKA Review plz. And https://github.com/UnUniFi/utils/pull/245

# Pull Request on some features of #40.

## Changes 
Multiple Nodes Support.
- Nodes are chosen at random from config.js
- Use configs array https://github.com/UnUniFi/utils/pull/245

### CODE REVIEW

File: projects/portal/src/app/models/config.service.ts [hunk 1]
```diff
@@ -36,14 +37,18 @@ export type Config = {
...
 })
 export class ConfigService {
   config: Config;
+  configs: Config[];
   constructor() {
-    this.config = config;
+    this.configs = configs;
+    this.config = configs[Math.floor(Math.random() * configs.length)];
+    this.configs = configs;
+    this.config = configs[Math.floor(Math.random() * configs.length)];
```
Comment (YasunoriMATSUOKA) [inline projects/portal/src/app/models/config.service.ts:52]: I guess 49, 50 and 51, 52 are duplicate. So you should delete one of them.

**Review by YasunoriMATSUOKA — CHANGES_REQUESTED** [review]
seems to be duplicate?

File: projects/portal/src/app/models/config.service.ts (hunk 1 above)
Comment (Senna46) [reply projects/portal/src/app/models/config.service.ts]: i fixed. Thank you.

**Review by YasunoriMATSUOKA — APPROVED** [review]
LGTM

**Pull request merged**